packages/types/news/6412.bugfix

```diff
@@ -0,0 +1 @@
+Some improvements and fixes in blocks and settings types @sneridagh
```

Comment:
Please specify, else this change log should just be deleted as it provides nothing informative to the reader.

```suggestion
Some improvements and fixes in blocks and settings types. @sneridagh
```